File: Back/src/MedicChat.Domain/model/VideoChat.cs
using System;
using MedicChat.Domain.model.Enums;

namespace MedicChat.Domain.model
{
    public class VideoChat
    {
        public int Id { get; set; }
        public string Relatorio { get; set; }
        public string Token { get; set; }
        public DateTime DataInicio { get; set; }
        public DateTime DataFim { get; set; }
        public VideoChatStatus EstadoVideoChat { get; set; }

        // Propriedade de Navegação!
        public int MedicoID { get; set; }
        public Medico Medico { get; set; }
        public int PacienteID { get; set; }
        public Paciente Paciente { get; set; }
    }
}

File: Front/MediChat-App/src/app/models/VideoChat.ts
import { Medico } from './Medico';
import { Paciente } from './Paciente';
import { VideoChatStatus } from './VideoChatStatus';

export interface VideoChat {
  id: number;
  relatorio: string;
  token: string;
  dataInicio: Date;
  dataFim?: Date;
  estadoVideoChat: VideoChatStatus;
  medicoID: number;
  medico: Medico;
  pacienteID: number;
  paciente: Paciente;
}
